### PR TITLE
MCS-397 MCS-402 MCS-408 Unity Optimizations

### PR DESCRIPTION
**Pull request by ThomasSchellenbergNextCentury**
The low-hanging-fruit of Unity runtime performance optimizations based on Ian's suggestions during our training.

MCS-397: Remove the unused class segmentation, normals, and flow cameras from the ImageSynthesis script.

MCS-402: Stop automatically generating a lot of visibility points on the walls, floor, and ceiling to reduce excessive raycasting. All structural objects are now always marked as visible in their object metadata output. (This seemed OK since "oracle" metadata just returns everything anyway.)

MCS-408: Replace mesh colliders with box colliders. (I also updated the crib, pacifier, and wardrobe prefabs in `mcs-private/UnityAssetStore/Custom`)

### CODE REVIEW

File: unity/Assets/ImageSynthesis/ImageSynthesis.cs [hunk 1]
```diff
@@ -24,10 +24,11 @@ public class ImageSynthesis : MonoBehaviour {
 	private CapturePass[] capturePasses = new CapturePass[] {
 		new CapturePass() { name = "_img" },
 		new CapturePass() { name = "_depth" },
-		new CapturePass() { name = "_id", supportsAntialiasing = false },
-		new CapturePass() { name = "_class", supportsAntialiasing = false },
-		new CapturePass() { name = "_normals"},
-		new CapturePass() { name = "_flow", supportsAntialiasing = false, needsRescale = true }, // (see issue with Motion Vectors in @KNOWN ISSUES)
+		new CapturePass() { name = "_id", supportsAntialiasing = false }
+        // MCS optimization: Remove all unused cameras
```
Comment (brianpippin) [inline unity/Assets/ImageSynthesis/ImageSynthesis.cs:28]: Should we remove the commented out code or are we saving it for some reason, if we are, perhaps add a TODO about why we are keeping it?

**Review by brianpippin — APPROVED** [review]

File: unity/Assets/ImageSynthesis/ImageSynthesis.cs (hunk 1 above)
Comment (ThomasSchellenbergNextCentury) [reply unity/Assets/ImageSynthesis/ImageSynthesis.cs:28]: You're right, I should update the comment there to say something like "uncomment these if we ever need them in the future"

**Review by rartiss55 — APPROVED** [review]
Looks good -- only thing that jumped out at me is that I ran this through the Python API, and noticed some structural objects set to {"visible": false} in my debug files, but there's likely some additional check on that end for visibility.

**Comment by ThomasSchellenbergNextCentury** [comment]
@rartiss55 Oh, that was a good catch. I forgot that there were both `visible` and `visibleInCamera` properties. It should be fixed now. Thanks!

**Review by rartiss55 — APPROVED** [review]

**Pull request merged**